Docker/python-nginx/python3.7-alpine/Dockerfile

```diff
@@ -0,0 +1,210 @@
+# Dockerfile written by Sebastian Ramirez <tiangolo@gmail.com> at https://github.com/tiangolo/uwsgi-nginx-docker
+
+FROM quay.io/cdis/python:3.7-alpine
```

Comment:
could you update [this readme](https://github.com/uc-cdis/cloud-automation/tree/a88b1e7326d2fdbdb67c94ae5daa8694aa11ed2a/Docker/python-nginx#readme)